src/SDKs/MachineLearningCompute/Management.MachineLearningCompute/generate.cmd

```diff
@@ -0,0 +1,18 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
```

Comment:
Please model this file according to the one that is already in other RPs to get an idea.

Reply:
Done.